sagemaker-python-sdk/dgl_lifesci_bap/README.md

```diff
@@ -0,0 +1,91 @@
+# Binding Affinity Prediction
```

Comment:
It looks like most of this readme content is copied from [DGL LifeSci's](https://github.com/yoheigon/dgl-lifesci/tree/master/examples/binding_affinity_prediction). Can you either simply link to the external readme, or more explicitly cite where this content is copied from?

Reply:
Yes, I simplified the README and cited the link to the external readme.